code/Testing.md

```diff
@@ -169,14 +169,16 @@ For documentation [`example.com`](https://www.iana.org/domains/reserved) can als
 
 Part of our [current implementation uses different _database backends_](Database), some of them are used to run the test in local without having to always depend of third party external systems when developing.
 
-Because using a different _database backend_ for development than in production can cause that some tests pass in local meanwhile fail using the production _database_ backend_, the CI runs all the tests with all the supported _backends_ or at least with the one used in production.
+Because using a different _database backend_ for development than in production can cause that some tests pass in local meanwhile fail using the production _database backend_, the CI runs all the tests with all the supported _backends_ or at least with the one used in production.
 
 Sometimes, meanwhile developing, it's less than ideal that for having feedback of each change the developer must push the code to run the CI, making pretty convenient to run the tests which the production _databse backend_. Currently this is the case for PostgreSQL in:
 
 * [Database migrations](Database).
 * Test SIM (i.e. `make test-sim`).
 
-In order to use PostgreSQL for running the tests in a local development machine you have to setup and run PostgreSQL v11 in your machine or run a Docker container using a [PostgreSQL image](https://hub.docker.com/_/postgres) and run the tests as follow:
+In order to use PostgreSQL<sup>1</sup> for running the tests in a local development machine you have to setup and run PostgreSQL v9.6 in your machine or run a Docker container using a [PostgreSQL image](https://hub.docker.com/_/postgres) and run the tests as follow:
 
 1. For _Go tests_, use the environment variable `STORJ_POSTGRES_TEST` to specify a connection URL (i.e. `postgres://[user][:password]@[host]?sslmode=disable`) when running them, for example `STORJ_POSTGRES_TEST="postgres://postgres:pass@localhost?sslmode=disable" go test ./...`
```

Comment:
Wondering whether we should rename `STORJ_POSTGRES_TEST` to `STORJ_TEST_POSTGRES` to be consistent with `STORJ_SIM`?

Reply:
Good to me.
I've created a ticket (https://storjlabs.atlassian.net/browse/V3-3106) because that change is out of the scope of this PR.